### PR TITLE
Add x5f2 schema

### PR DESCRIPTION
**Pull request by michele-brambilla**

### CODE REVIEW

File: tests/test_x52f.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import pytest
+from streaming_data_types.status_x5f2 import serialise_x5f2, deserialise_x5f2
+
+
+class TestEncoder(object):
+
+    def test_serialises_and_deserialises_x5f2_message_correctly(self):
+        #     """
+        #     Round-trip to check what we serialise is what we get back.
+        #     """
```
Comment (matthew-d-jones) [inline tests/test_x52f.py:10]:
```suggestion
        """
        Round-trip to check what we serialise is what we get back.
        """
```

File: tests/test_x52f.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+        original_entry = {
+            "software_name" : "nicos/test",
+            "software_version" : "1.0",
+            "service_id" : "1a2b3c",
+            "host_name" : "//localhost",
```
Comment (matthew-d-jones) [inline tests/test_x52f.py:16]: This is just host name, not any kind of full URI, so not the place for "//" I think.
```suggestion
            "host_name" : "localhost",
```

File: tests/test_x52f.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        assert entry.process_id == original_entry["process_id"]
+        assert entry.update_interval == original_entry["update_interval"]
+        assert entry.status_json == original_entry["status_json"]
+
+    def test_if_buffer_has_wrong_id_then_throws(self):
+        original_entry = {
+            "software_name" : "nicos/test",
+            "software_version" : "1.0",
+            "service_id" : "1a2b3c",
+            "host_name" : "//localhost",
```
Comment (matthew-d-jones) [inline tests/test_x52f.py:38]:
```suggestion
            "host_name" : "localhost",
```

File: tests/test_x52f.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+class TestEncoder(object):
+
+    def test_serialises_and_deserialises_x5f2_message_correctly(self):
+        #     """
+        #     Round-trip to check what we serialise is what we get back.
+        #     """
+
+        original_entry = {
+            "software_name" : "nicos/test",
+            "software_version" : "1.0",
```
Comment (matthew-d-jones) [inline tests/test_x52f.py:14]: We are using MAJOR.MINOR.PATCH versioning, so may as well make the example match:
```suggestion
            "software_version" : "1.0.0",
```

File: tests/test_x52f.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        assert entry.service_id == original_entry["service_id"]
+        assert entry.host_name == original_entry["host_name"]
+        assert entry.process_id == original_entry["process_id"]
+        assert entry.update_interval == original_entry["update_interval"]
+        assert entry.status_json == original_entry["status_json"]
+
+    def test_if_buffer_has_wrong_id_then_throws(self):
+        original_entry = {
+            "software_name" : "nicos/test",
+            "software_version" : "1.0",
```
Comment (matthew-d-jones) [inline tests/test_x52f.py:36]:
```suggestion
            "software_version" : "1.0.0",
```

File: requirements.txt [hunk 6]
```diff
@@ -1,2 +1,2 @@
-flatbuffers
+flatbuffers>=1.12
```
Comment (mattclarke) [inline requirements.txt:1]:
```suggestion
flatbuffers>=1.11
```

**Pull request merged**